core/primitives/src/sr25519.rs

```diff
@@ -476,29 +476,22 @@ impl TraitPair for Pair {
 
 	/// Verify a signature on a message. Returns true if the signature is good.
 	fn verify<P: AsRef<Self::Public>, M: AsRef<[u8]>>(sig: &Self::Signature, message: M, pubkey: P) -> bool {
-		let signature: schnorrkel::Signature = match schnorrkel::Signature::from_bytes(&sig.as_ref()) {
-			Ok(some_signature) => some_signature,
-			Err(_) => return false
-		};
+		// Match both schnorrkel 0.1.1 and 0.8.0+ signatures, supporting both wallets
```

Comment:
I think we should open an issue for this? So we don't forget to switch.

Reply:
Probably.  And/or on schnorrkel.  I'll remove the preaudit_deprecated feature entirely whenever all testnets migrate.

Reply:
Okay, yeah then let's add this issue to schnorrkel.

Reply:
https://github.com/w3f/schnorrkel/issues/42